doc/additional_commands.rst

```diff
@@ -67,7 +80,21 @@ Here are the parameters for poller mode:
 Detect changes and submit them automatically
 --------------------------------------------
 
-.. TODO: Add description
+Unlike default mode, Universum in `submit` mode **requires an already prepared local repository.** For example:
+
+* In case of Git:
+
+    - the repo should be already cloned
+    - the required branch should be already checked out
+
+* In case of P4:
+
+    - the client should be already created
+    - the directory should be already synced
+    - all the required shelves should be applied
```

Comment:
Shelves? Did you mean changes?

Reply:
Well, in terms of Perforce I meant exactly shelves that should be unshelved if needed. _Before_ applying the actual "manual" changes.